packages/backend/src/api-doc.ts

```diff
@@ -2,52 +2,10 @@ import {
   OpenAPIRegistry,
   OpenApiGeneratorV3,
 } from '@asteasolutions/zod-to-openapi'
-import {
-  Entity,
-  Schema,
-  Invitation,
-  InvitationDto,
-  Submission,
-} from '@ssi-trust-registry/common'
 import { z } from 'zod'
+import { Context } from './server'
```

Comment:
This creates a kind of cyclic dependency, from `api-doc` to `server` and vice-versa.